openvdb/openvdb/python/test/TestOpenVDB.py

```diff
@@ -728,12 +718,7 @@ def createArrays():
 
     def testMeshConversion(self):
         import time
-
-        # Skip this test if NumPy is not available.
-        try:
-            import numpy as np
-        except ImportError:
-            return
+        import numpy as np
```

Comment:
If we assume that numpy is available, we can move these imports to the top of the file.